test/karma.tests.js

```diff
@@ -8,6 +8,27 @@ import { unstable_resetCreateSelectorCache } from '@mui/x-data-grid/internals';
 const packagesContext = require.context('../packages', true, /\.test\.tsx$/);
 packagesContext.keys().forEach(packagesContext);
 
+// See https://github.com/mui/mui-x/pull/3607#discussion_r784785626
+const jsonStringify = JSON.stringify;
```

Comment:
TODO: move it to https://github.com/mui/material-ui/blob/master/test/utils/setupKarma.js

Reply:
No, it won't work in `material-ui` repo